influunt-app/app/views/directives/influunt-tabs/_tab.html

```diff
@@ -1,5 +1,5 @@
 <li>
-  <a data-toggle="tab" href="#tabs-1" aria-expanded="false">{{ 'controladores.anel' | translate }} {{ anel.posicao }}</a>
+  <a data-toggle="tab" href="#{{ nameTabs }}-{{ anel.posicao }}" aria-expanded="false">{{ nameTabs }} {{ anel.posicao }}</a>
```

Comment:
Se estivéssemos usando as tabs da forma que elas foram feitas p/ usar, esse atributo indica qual elemento vai ficar visível quando a tab for selecionada. Como não estamos usando, não faz diferença essa troca.